src/icons/index.tsx

```diff
@@ -18,12 +16,12 @@ import {
   ArrowDown as ArrowDownIcon,
   ArrowRight as ArrowRightIcon,
   ArrowUp as ArrowUpIcon,
+  ArrowUpRight as LinkIcon,
```

Comment:
Unused - remove?

Reply:
this is still used, i'm just switching the underlying icon